Zend/zend_compile.h

```diff
@@ -349,6 +349,9 @@ typedef struct _zend_oparray_context {
 /* call through internal function handler. e.g. Closure::invoke() */
 #define ZEND_ACC_CALL_VIA_HANDLER     ZEND_ACC_CALL_VIA_TRAMPOLINE
 
+#define ZEND_SHORT_CIRCUITING_CHAIN_ISSET (1 << 0)
+#define ZEND_SHORT_CIRCUITING_CHAIN_EMPTY (1 << 1)
```

Comment:
This isn't a bit mask but looks like it, so I would suggest to make this three constants 0/1/2.